Sources/Extensions/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -180,6 +180,25 @@ public extension Sequence where Element: Equatable {
         return true
     }
 
+    /// SwifterSwift: Check if sequence contains element exactly `times` times
+    ///
+    ///     [1, 2, 3, 1].contains(1, times: 1) -> false
+    ///     [1, 2, 3, 1].contains(1, times: 2) -> true
+    ///     ["h", "e", "l", "l", "o"].contains("l", times: 2) -> true
+    ///
+    /// - Parameters:
+    ///     - element: element to check
+    ///     - times: amount of times element should appear in the sequence
+    /// - Returns: true if sequence contains element `times` times
+    public func contains(_ element: Element, times: Int) -> Bool {
+        var matches = 0
```

Comment:
You can break this all down to 1 line:
```Swift
return filter { $0 == element }.count == times
```

Reply:
Actually, `filter` is a bad choice as it will create another array. Better to use `reduce`. I.e.
```Swift
public func count(_ element: Element) -> Int {
    return reduce(0) { ($1 == element) ? $0 + 1 : $0 }
}
```